src/main/java/games/battlelore/cards/CommandCard.java

```diff
@@ -0,0 +1,125 @@
+package games.battlelore.cards;
+
+import core.components.Card;
+import games.battlelore.components.Unit;
+import games.coltexpress.cards.ColtExpressCard;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CommandCard extends Card {
+
+    //Command cards allow different type of "orders" to troops
+    public enum CommandType {
+        LineAdvance, //1-1-1
+        PatrolLeft, //2-0-0
+        AttackRight, //0-0-3
+        Wedge, // 1-2-1
+        Onslaught,
+        DarkenTheSkies,
+        BattleMarch
+    }
+
+    public enum Effects {
+        LoreGain,// BattleLore
+        AdditionalMove, //Onslaught
+        AdditionalAttack
+        //DarkenTheSkies
+    }
+
+    public enum Condition {
+        NotWeak, //BattleMarch
+        OnlyInfantry //Onslaught
+    }
+
+    private int orderCount;
+    private int bonusAttackCount;
+    private int bonusLorePoints;
+    private int bonusMoveCount;
+    private Boolean hasCondition;
+    protected String title;
+    protected String explanation;
+
+    public CommandCard(String title, String explanation, int orderCount, int bonusAttackCount, int bonusLorePoints, int bonusMoveCount) {
```

Comment:
The constructors here should all call a super() constructor since you are extending another class.